### PR TITLE
CORE-13994. Start menu, shortcuts, r61988: update untranslated strings too

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

I am not sure how translations are maintained. (Translators should re-check r61988...)
By default, I suggest to update all older string occurrences. (To improve consistency.)

JIRA issue: [CORE-13994](https://jira.reactos.org/browse/CORE-13994)

## Proposed changes

- Update (apparently) untranslated strings to their new default.

### CODE REVIEW

**Comment by binarymaster** [comment]
@SergeGautherie why you're using so long branch names? :octocat:

**Comment by SergeGautherie** [comment]
Shell: @yagoulas

**Comment by SergeGautherie** [comment]
> why you're using so long branch names?

To be "explicit". Why not?

File: media/inf/shortcuts.inf [hunk 1]
```diff
@@ -287,7 +287,7 @@ CALC_TITLE=Rechner
 CALC_DESC=Starte Taschenrechner
 NOTEPAD_TITLE=Editor
 NOTEPAD_DESC=Starte Texteditor
-MSTSC_TITLE=Remote Desktop
+MSTSC_TITLE=Remote Desktop Connection
```
Comment (ThFabba) [inline media/inf/shortcuts.inf]: "Remote Desktop" is an acceptable translation, so either keep it or use Verbindung as found in the line below.

Comment (ThFabba) [reply media/inf/shortcuts.inf]: Rest of the changes lgtm.

Comment (SergeGautherie) [reply media/inf/shortcuts.inf]: > use Verbindung

Done.

**Pull request merged**